config.go

```diff
@@ -296,7 +296,7 @@ func (c *config) discoverSingle(glob string) (map[string]string, error) {
 		}
 
 		// If the filename has a ".", trim up to there
-		if idx := strings.Index(file, "."); idx >= 0 {
+		if idx := strings.Index(file, ".exe"); idx >= 0 {
```

Comment:
Can plugins be `.exe` for other OS besides windows? 
If this is only for windows, there's a if statement that excludes everything that is not `.exe` before this. In that case, the file will always be `.exe`.

Reply:
I think .exe is an only-windows thing. And windows files _must have .exe

Reply:
I thought this was for making sure is a exe but is actually looking specifically for the file name, right? It can be an exe but without .exe in the name. 🤔 now I get

Reply:
oh yeah this is saying "it's cool if the builder has .exe on the end, but it can't have any other extension"